services/app-api/src/jwt/jwt.ts

```diff
@@ -45,6 +47,8 @@ interface JWTLib {
 
 function newJWTLib(config: JWTConfig): JWTLib {
     return {
+        // this is an experiment, using `curry` here, It seems clean but I'm not sure
+        // exactly what it's getting us yet -wml
```

Comment:
My understanding is currying is good for higher order functions that are functions that return other functions